charts/newrelic-prometheus-agent/static/curatedexperience.yaml

```diff
@@ -0,0 +1,6 @@
+list_regexes:
```

Comment:
Will this list expand automatically on the user's effective config when the chart upgrades?

Reply:
not sure I am following 😄 

Are you wondering about the upgrade process?

Reply:
This will be similar to any configuration change, no difference